harbor/tests/common.py

```diff
@@ -17,6 +17,7 @@
     ('harbor.projects.count', False),
     ('harbor.disk.free', True),
     ('harbor.disk.total', True),
+    ('harbor.registry.read_only', False),
```

Comment:
I think you should be able to do something like this here:

```python
HARBOR_METRICS = [
    # Metric_name, requires admin privileges
    ('harbor.projects.count', False),
    ('harbor.disk.free', True),
    ('harbor.disk.total', True),
]
if HARBOR_VERSION >= VERSION_1_5:
    HARBOR_METRICS.append(('harbor.registry.read_only', False))
```

But you'll have to move the block a few lines below so that 'HARBOR_VERSION' is defined.